ONSdigital/sdc-global-design-patterns
# Panel content margin bottom

### Context of this PR

Paragraphs and lists within a panel carry `margin-bottom: 1rem` and this produces compound space when within a panel. This fix sets the margin-bottom to zero using last-of-types for paragraph and lists.

### How to test

- Pull this branch and run `yarn start`.
- Browse to the panel component and view the simple info panel.
- Add additional paragraphs or lists to a panel and check the appearance is as expected.

### Fixes
- Issue #52
